### PR TITLE
Add an svn folder upgrade utility

### PR DESCRIPTION
**Pull request by tomjn**
The core provisioner does this, but it's only really applicable to a very small number of people, so lets put it in a utility and remove it from a future version so it isn't mandatory

### CODE REVIEW

File: svn-folder-upgrade/provision.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+echo " * Running all PHP provisioners"
```
Comment (msaggiorato) [inline svn-folder-upgrade/provision.sh:3]: Does this need to be here?

**Comment by tomjn** [comment]
Yes but with a different message

File: svn-folder-upgrade/provision.sh (hunk 1 above)
Comment (Mte90) [reply svn-folder-upgrade/provision.sh:3]:
```suggestion
echo " * Upgrading SVN to Git in progress"
```

Comment (tomjn) [reply svn-folder-upgrade/provision.sh:3]: This doesn't migrate  svn to git, it upgrades SVN repos from the old format to  the new repo format, via `svn upgrade`.

I thought I had changed this but seems I missed it, good catch

**Comment by Mte90** [comment]
The pr to remove that code on VVV is already avalaible?

**Comment by tomjn** [comment]
> The pr to remove that code on VVV is already avalaible?

No, and it shouldn't be a blocker

**Comment by Mte90** [comment]
So we keep that code on VVV and do also a utility? It isn't a duplicate code?
In the first ticket it is written to remove on future version so it is the case to open a ticket just to remind us when is time to do it.

**Review by Mte90 — APPROVED** [review]

**Comment by tomjn** [comment]
> So we keep that code on VVV and do also a utility? It isn't a duplicate code?

Yes but I've added a PR to remove it from VVV core. Even then, having it in core isn't a blocker for having it in a utility. E.g. PHP 7.2. Also even if we remove it in the next version, older instances of VVV will have this utility on reprovision. It won't run unless they add it though

**Pull request merged**